app/src/main/java/org/apache/fineract/ui/online/customers/customerlist/CustomersFragment.java

```diff
@@ -271,14 +271,15 @@ private void setUpSearchInterface(Menu menu) {
         searchView.setOnQueryTextListener(new SearchView.OnQueryTextListener() {
             @Override
             public boolean onQueryTextSubmit(String query) {
-                findCustomer(query);
                 return false;
             }
 
             @Override
             public boolean onQueryTextChange(String newText) {
                 if (TextUtils.isEmpty(newText)) {
                     customerAdapter.setCustomers(customers);
+                } else {
+                   findCustomer(newText);
```

Comment:
Here you have check style violation. You have added 1 more space on line number 283.
How to solve it?
Go to your CustomersFragment.java file enter `Ctr+Alt+SPACE`. It will solve the problem.